cypress/integration/App.spec.js

```diff
@@ -1,5 +1,9 @@
 import '@testing-library/cypress/add-commands';
-import { getCurrentMonth, getUKFormattedEuros } from '../../src/utils/helpers';
+import {
+  getCurrentMonth,
+  getUKFormattedEuros,
+  getUKFormattedDate,
+} from '../../src/utils/helpers';
```

Comment:
and here is the prettier again :)

do you find this formatting appealing?

Reply:
Not really. But I'm not sure what we can do about it. Maybe increase the col width?

I'm really on the fence at the moment. I don't like how Prettier formats, but I do really like how it provides easy consistency.